lib/CL/devices/tbb/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#=============================================================================
+#   CMake build system files
+#
+#   Copyright (c) 2014 pocl developers
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a copy
+#   of this software and associated documentation files (the "Software"), to deal
+#   in the Software without restriction, including without limitation the rights
+#   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#   copies of the Software, and to permit persons to whom the Software is
+#   furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included in
+#   all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#   THE SOFTWARE.
+#
+#=============================================================================
+
+add_compile_options("-std=c++11")
```

Comment:
if you did this, this is ugly in multiple ways (globally set an option, and in a non-standard way as different compilers might have different options)

better set per target in a portable way, e.g.:
https://github.com/noma/typa/blob/513154f1d85a8e8baa16afefd051dc351dc45b52/CMakeLists.txt#L22

Reply:
Yes, I did this by [copying from the accel device](https://github.com/new2f7/pocl/blob/master/lib/CL/devices/accel/CMakeLists.txt#L26). Should I fix this for both devices or just for tbb?

Reply:
if pocl-style fix is unclear, make a small issue at pocl for the accel device, otherwise just fix and do a separate pull request for accel

Reply:
Commit that fixes this issue for accel on the master branch: https://github.com/new2f7/pocl/commit/7b1e653522fab29565350d173c054618f0a2bb0a